assets/panel/debugger.properties

```diff
@@ -104,6 +104,10 @@ pauseOnExceptionsItem=Pause on exceptions
 # when the debugger will not pause on any caught exception
 ignoreCaughtExceptionsItem=Ignore caught exceptions
 
+# LOCALIZATION NOTE (pauseOnCaughtExceptionsItem): The pause on exceptions checkbox description
+# when the debugger should puase on caught exceptions
```

Comment:
typo: puase

Reply:
WHY DO I KEEP DOING THAT?!  Seriously, I did that like 100 times while working on this stuff.  Making me so mad.